courses/models.py

```diff
@@ -301,6 +309,18 @@ def first_unexpired_run(self):
             lambda course_run: course_run.is_unexpired,
         )
 
+    @property
+    def unexpired_run_dates(self):
```

Comment:
Sorry, one more thing. This is returning unexpired `CourseRun`s, not dates, so this should be called `unexpired_run` not `unexpired_run_dates`. Feel free to merge after this change is made, tests pass, and you confirm that the UI still works